consensus/src/detail/spend_cache.rs

```diff
@@ -165,13 +176,86 @@ impl<'a> CachedInputs<'a> {
         Ok(())
     }
 
+    fn verify_signatures(&self, tx: &Transaction) -> Result<(), BlockError> {
+        for (input_idx, input) in tx.get_inputs().iter().enumerate() {
+            let outpoint = input.get_outpoint();
+            let prev_tx_index_op = self.get_from_cached(outpoint)?;
+
+            let tx_index = prev_tx_index_op
+                .get_tx_index()
+                .ok_or(BlockError::PreviouslyCachedInputNotFound)?;
+
+            use common::chain::SpendablePosition;
```

Comment:
There is a clippy lint about putting `use` items in the middle of a function. We can disable it if preferred.

Reply:
In CI clippy seems to be happy.

Reply:
I see it here on Github:

```
GitHub Actions
/ clippy
adding items after statements is confusing, since items exist from the start of the scope
warning: adding items after statements is confusing, since items exist from the start of the scope
   --> consensus/src/detail/spend_cache.rs:188:13
    |
188 |             use common::chain::SpendablePosition;
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: requested on the command line with `-W clippy::items-after-statements`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#items_after_statements
```

Reply:
Alrighty. I'll move it.